src/gateway/reference/proxy.md

```diff
@@ -1269,6 +1269,8 @@ This makes {{site.base_gateway}} forward the `Connection` and `Upgrade` headers
 upstream service, instead of dismissing them due to the hop-by-hop nature of a
 standard HTTP proxy.
 
+If you are using either of the [WebSocket Size Limit](pluginlink) or the [WebSocket Validator](pluginlink) plugins, this may slow the performance of the proxy WebSocket traffic.
```

Comment:
@cloudjumpercat I need to fill out this section some more to explain the differences between service/route protocols (http/https/ws/wss) for WebSocket traffic and when one might use one or the other, _including_ a call-out that ws/wss services are an EE-only feature.

I'm holding off on adding that for the moment in case you are still doing some more branch maintenance that would require a rebase/force-push. Can you @ me when you're ready for me to add more changes to the branch?

Reply:
@flrgh Okay, I should be done with editing branches for now (just had one comment on a placeholder value). I'll probably revisit this tomorrow to see if I can add to the PDK/Admin API doc.